src/builder/QuarkBuilder.sol

```diff
@@ -71,6 +75,12 @@ contract QuarkBuilder {
         uint256 blockTimestamp;
     }
 
+    struct BridgePayload {
+        uint256 amountLeftToBridge;
+        uint256 bridgeActionCount;
+        uint256 amountToBridge;
+    }
+
```

Comment:
By creating struct to hold just 3 more variables, we could avoid from having to use --via-ir.  
But downside is there is a struct that wrap holds multiple variables, might impact the readability of the code as people are used to initialize and declare variables by its own instead of a wrapper struct.

Reply:
I'd say let's not resort to this for now, since it does hurt readability. There also might be other ways to do this that are cleaner, like block scoping.

Reply:
Ok 👍